android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -580,10 +599,11 @@ private int stop() {
 
     @ReactMethod
     public void stop(Promise promise) {
-        if (notReady(promise)) return;
-
-        int result = stop();
-        boolean resultValue = (result == TextToSpeech.SUCCESS) ? Boolean.TRUE : Boolean.FALSE;
+        boolean resultValue = Boolean.TRUE;
```

Comment:
I'll leave it up to you, but these are not really Java coding standards.
I'd use `true` rather than `Boolean.TRUE`
and
`resultValue = result == TextToSpeech.SUCCESS`

Reply:
I kept the same style as was used in the code it came from / already here and elsewhere in the original source. I'll make a change over the whole file then.

Reply:
I'll also change ready from Boolean to boolean to remove the null option, just make it to confusing in the checks

Reply:
I suspected that it was to stay consistent with what was used before. I can live with it....
Just thought it looks weird.

Reply:
I'll change it to just a true / false boolean as that would also have made the mistake below less like to happen

Reply:
rework ready status in commit: [d324968](https://github.com/tomtom-international/react-native-tts/pull/2/commits/d324968c7354148a8f7405b9376005661508e879)